tlslite/tlsconnection.py

```diff
@@ -773,6 +776,9 @@ def _clientGetServerHello(self, settings, clientHello):
                         AlertDescription.illegal_parameter,
                         "Server selected ALPN protocol we did not advertise"):
                     yield result
+        if settings.useHeartbeatExtension and \
+                serverHello.getExtension(ExtensionType.heartbeat):
```

Comment:
> How can I get, if the mode, what I got from opposite site, is in HeartbeatMode class?

it if isn't `toRepr()` will return `None`, but since there can be just two values here, each having a different code-path associated with it, having a default `else` case should be both enough and cleaner